lib/kubernetes-deploy/kubectl.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 module KubernetesDeploy
+  MIN_KUBE_VERSION = '1.6.0'
```

Comment:
I'd suggest putting this directly in `kubernetes-deploy.rb` since you're defining it on `KubernetesDeploy`, not on `KubernetesDeploy::Kubectl`.